GitUI/CommandsDialogs/RevisionDiffControl.Designer.cs

```diff
@@ -305,7 +305,7 @@ private void InitializeComponent()
             this.diffDeleteFileToolStripMenuItem.Image = global::GitUI.Properties.Images.DeleteFile;
             this.diffDeleteFileToolStripMenuItem.Name = "diffDeleteFileToolStripMenuItem";
             this.diffDeleteFileToolStripMenuItem.Size = new System.Drawing.Size(296, 22);
-            this.diffDeleteFileToolStripMenuItem.Text = "Delete file";
+            this.diffDeleteFileToolStripMenuItem.Text = "Delete file(s)";
```

Comment:
I would like to remove (s) in all situations as it is used inconsistently and is hard to translate, so I prefer to not add another location
..and this will affect the translation, not sure why it is not detected...

Reply:
Have a look at `ResourceManager.Strings` - we have examples of single/multiple formatted strings. Please use the same mechanism.

Reply:
still open

Reply:
This could be shared with FormCommit (maybe FormResolveConflicts), no requirement to do so for me